ush/atmos_extractvars.sh

```diff
@@ -72,10 +72,15 @@ for outtype in "f2d" "f3d"; do
 
     for infile in "${infile1}" "${infile2}"; do
       if [[ -f "${infile}" ]]; then # check if input file exists before extraction
+        new_infile="${outdirpre}/$(basename "${infile}")_ext"
+        if ! cpfs "${infile}" "${new_infile}"; then
+          echo "ERROR: Failed to copy ${infile} to ${new_infile}. Skipping."
```

Comment:
```suggestion
          echo "FATAL ERROR: Failed to copy ${infile} to ${new_infile}. Skipping."
```
`FATAL ERROR` or `ERROR` should result in an immediate exit.  Perhaps this is worthy of a `WARNING`.